beacon_node/src/cli.rs

```diff
@@ -816,6 +816,15 @@ pub fn cli_app() -> Command {
                 .action(ArgAction::Set)
                 .display_order(0)
         )
+        .arg(
+            Arg::new("state-cache-max-MiB")
```

Comment:
Capitalisation in a flag name is very unusual. We should use either `-mb`, or just `state-cache-max-size` and the description can state that the quantity is in megabytes.

```suggestion
            Arg::new("state-cache-max-mb")
```